src/docs/product/integrations/integration-platform/index.mdx

```diff
@@ -6,374 +6,107 @@ redirect_from:
-- Actions taken in Sentry will connect to this user rather than the integration/app. For example, if your alerting integration automatically assigns issues to a teammate, it will appear in the history as “Alice assigned Bob issue XYZ” rather than “&lt;Your Sentry Integration apps&gt; assigned Bob issue XYZ.”
+- If this user leaves the organization or their permissions change, the integration can break. A Sentry integration, on the other hand, is a separate, independent entity with its own permissions.
+- OAuth apps and auth tokens permit the user to access other Sentry organizations they belong to, so there is limited isolation.
+- Actions taken in Sentry will connect to this user rather than the integration/app. For example, if your alerting integration automatically assigns issues to a teammate, it will appear in the history as “Alice assigned Bob issue XYZ” rather than “&lt;Your Sentry integration&gt; assigned Bob issue XYZ.”
 
 **Managing Webhooks**
-
-With Sentry Integration apps you can manage webhooks in the UI, as opposed to the current state where you need to make API calls to create/update/delete/retrieve webhooks. The latter is not only more cumbersome but also harder to keep track of and maintain visibility across your team.
+With Sentry integrations you can manage webhooks in [sentry.io](https://sentry.io), as opposed to the current state where you need to make API calls to create/update/delete/retrieve webhooks. This is not only more cumbersome, but also harder to keep track of and makes it difficult to maintain visibility across your team.
 
 **Scope**
+Currently, webhooks must be configured per-project, which is added effort especially when adding/removing projects. With Sentry integrations, you’ll receive notifications for all projects in your organization.
 
-Currently, webhooks must be configured per-project, which is unnecessary overhead especially when adding/removing projects. With Sentry Integration apps, you’ll receive notifications for all projects in your organization.
-
-**New Event Types**
-
-Sentry Integration apps expose a richer set of events that are not available using the legacy service hooks API. Currently, this includes the following webhooks:
+**Event Types**
+Sentry integrations expose a richer set of events that are not available using the legacy service hooks API. Currently, this includes the following webhooks:
 * Issue Created
 * Issue Resolved
 * Issue Ignored
 * Issue Assigned
 * Comment Created
 * Comment Updated
 * Comment Deleted
+* Error Created
 
-In the future, we’ll add more types of webhooks to Sentry Integration apps.
-
-**Custom UI Components**
+In the future, we’ll add more types of webhooks to Sentry integrations.
 
-Sentry Integration apps will be able to augment Sentry’s UI in meaningful and feature-rich ways. For example, on the issue page, you could have a button that creates a task in your task-management tool, or open a line in the stack trace in another tool.
+## Learn More
```

Comment:
I really like this learn more section but I'm wondering if we should be calling attention to the subpages for Internal Integrations and Public Integrations earlier some how.

It's probably fine if we trim down this page a bit, but as is I think they appear too low in the page to actually get the visibility they need.